src/testswarm-browserstack.js

```diff
@@ -374,11 +374,10 @@ self = {
 					// "Android Browser" (even the ones that use "Chrome Mobile" in Android 4.4+).
 					// Instead of mapping all this (which would require complicating mapHelper to
 					// map browserFamily "Chrome" to "Android Browser", if osFamily is "Android"),
```

Comment:
The comment above here says something about the theoretical mapping (which you seem to add in this commit) being "complicated". In particular that it would need to be limited to "Android" OS to avoid breaking other devices/OS that use "Chrome Mobile", which might become inaccessible as-is due to "Android Browser" unlikely to be accepted for those non-Android devices.

Can you look into that and amend this comment or the code accordingly?

Reply:
I see. This might indeed be a problem in the future; for now BrowserStack only shares Safari on iOS & Chrome Mobile on Android so Chrome => Android.

If we want to be ready for when/if BrowserStack adds Chrome on iOS, the logic needs to be more complex, indeed.

The current (on `master`) condition is quite strange, though - the comment mentions iOS & Android but the condition only matches Android and only when both `osMajor` & `osMinor` are specified - that last part is what caused the issue I had here.